### DISCUSSION TITLE
Guys, im trying to get achievements

### DISCUSSION THREAD

**Discussion by isro-ori** [community/community#65800] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Anyone can tell me how to get yolo badge?

Thx in advance.💖

**Comment by diegotorrent** [comment]
You need create a branch, attach a file, merge the branchs and make pull request.

=)

**Comment by isro-ori** [comment]
Thank you

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
